editor/metaboxes/style.scss

```diff
@@ -0,0 +1,94 @@
+/**
+ * These are the styles that are not loaded from within the iframe.
+ */
+
+.gutenberg-metabox-iframe {
+	width: 100%;
+	position: relative;
+
+	&.closed {
+		display: none;
+	}
+
+	.loading-overlay {
+		background-color: rgba( 0, 0, 0, 0.6 );
+		display: none;
+		position: absolute;
+		width: 100%;
+		height: 100%;
+		top: 0;
+		left: 0;
+		z-index: 1;
+
+		&.visible {
+			display: flex;
+			align-items: center;
+			justify-content: center;
+		}
+
+		.loading-overlay__text {
+			color: $white;
+			font-weight: 700;
+		}
+	}
+
+	.iframe--updating {
```

Comment:
We prefer `is-modifier` classes like `is-updating`.

Reply:
Okay, no problem, will change..

Reply:
I need to probably review the CSS guidelines, and retouch a couple of things within the iframe as well.